<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ZDoList</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>373</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionNew_task"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuSort_by">
     <property name="title">
      <string>Sort by ...</string>
     </property>
     <addaction name="actionsortcontext"/>
     <addaction name="actionsortpriority"/>
     <addaction name="actionsortdue_date"/>
     <addaction name="actionsortpriority_days_left"/>
    </widget>
    <addaction name="menuSort_by"/>
    <addaction name="actionFilter_nonprioritized"/>
    <addaction name="separator"/>
    <addaction name="actionCollapsed"/>
    <addaction name="actionExpanded"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNew_context">
   <property name="text">
    <string>&amp;New context</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="text">
    <string>New &amp;project</string>
   </property>
  </action>
  <action name="actionNew_task">
   <property name="text">
    <string>N&amp;ew task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionFilter_nonprioritized">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter nonprioritized</string>
   </property>
  </action>
  <action name="actionGroup_Projects">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Group Projects</string>
   </property>
  </action>
  <action name="actionCollapsed">
   <property name="text">
    <string>Collapse</string>
   </property>
  </action>
  <action name="actionExpanded">
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Expand</string>
   </property>
   <property name="toolTip">
    <string>Expand</string>
   </property>
  </action>
  <action name="actionsortcontext">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>context</string>
   </property>
  </action>
  <action name="actionsortpriority">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>priority</string>
   </property>
  </action>
  <action name="actionsortdue_date">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>due date</string>
   </property>
  </action>
  <action name="actionsortpriority_days_left">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>priority/days left</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
